api/go.sum

```diff
@@ -1,4 +1,3 @@
-cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Is removing these dependencies related to the rest of this commit?

Reply:
found that running `go mod tidy` for each commit makes more (not most) sense to keep things clean. Otherwise there will be random commits with this. Please suggest a better workflow